applications/slonigiraf.md

```diff
@@ -0,0 +1,186 @@
+# W3F Grant Proposal
+
+- **Project Name:** Slonigiraf
+- **Team Name:** Slonigiraf
+- **Payment Address:** bc1qyuk4rdeqvc2rmetfhr3key4clty5sglaw4n5wx (BTC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+- Slonigiraf is an p2p education system.
+- We want to bring to Web3 infrastructure the renowned Slonigiraf learning game, which has been designed to deliver more
+  interactive and productive lessons through peer-to-peer learning and quality control through cross-exams. This game in
+  a nutshell consists of the following parts:\
+  Sara (student) pays Tanya (teacher) to teach skill_1 and issue a certificate that Sarah has this skill. Such a
+  certificate not only shows that Sarah has skill_1, but also gives Sarah the ability to allow other people fine Tania a
+  fixed amount of money if Sarah lacks the skill. The penalty could be applied only once.\
+  Sarah then goes to another teacher Teri and pays him to teach skill_2 and get a certificate. Sarah provides Teri the
+  opportunity to revise skill_1 and penalize Tania.\
+  Sarah then goes to Ed (the employer) to get a job that requires skill_1 and skill_2. Sarah presents Ed with
+  certificates and give Ed a chance to punish Tanya and Terry. If Ed believes Sarah has the skills, he will hire Sarah
+  and also pay Sarah for the certificates use. Otherwise, Ed may penalize teachers for incorrect certificates.\
+  SLON works as a reputation token. You could be surprised that we provide a possibility to buy reputation but this design
+  really works - since you pay for your reputation, you value it.\
+  The relation between SLONs and guarantees is following: teacher can issue 5 guarantees having 100 SLONs on his account, 
+  but each guarantee, if penalized, costs teacher 100 SLON.\
+  All parties: Sarah, Tanya, Teri and Ed are economically motivated to behave honestly.\
+  Ed is economic incentivized penalize a teacher in case if Sarah lack the skill because he gets SLON from such teacher.\
+  But also Ed can't behave dishonestly. If he starts to penalize teachers in case when skill is present - he will lose
+  all new workers - nobody whants to work with unfair hirer.\
+  In different professions two models of payments exist: per hour (or per month) and per result. The first one is secure to workers but doesn't motivate them to produce the result. The second one - paying for the result enables to democratize the service market - you just pay for what you need and don't require CVs of worker. Also it enables us to include students and pupils in system as teachers - students teach each other and pay each other only for the result.\
```

Comment:
I support what you're aiming to achieve here, but feel that a concrete design of incentivizing the correct behavior / punish misbehavior is missing.

As mentioned earlier, a parachain seems like a bad fit for this project, given the cost in securing one.

As such, I do not support the application in its current state.